realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -87,7 +87,9 @@ void BaseRealSenseNode::publishTopics()
     setupDevice();
     setupPublishers();
     setupStreams();
-    publishStaticTransforms();
+    if (_enable[DEPTH]) {
+      publishStaticTransforms();
```

Comment:
We want tf for enabled streams to be published, such as color_frame to camera_link even if depth is disabled.

Reply:
Okay, the publishStaticTransforms() function would have to be refactored so the other tf frames are independent of the depth frame_id. They are currently coupled to the extrinsics of the depth camera, which IIRC, are not available if the depth camera is disabled, causing a crash.